xcresult/Model/ResultHandler.swift

```diff
@@ -15,7 +15,7 @@ struct ResultHandler {
     // MARK: Internal
     
     /// Handles a URL, expecting the URL to be a path to an xcresult directory.
-    func handle(resultURL url: URL) {
+	func handle(resultURL url: URL, shouldOpenResultFile: Bool) {
```

Comment:
Do you use 4 spaces or two? I use 4 and not sure what this happened. 🤔

Reply:
<img width="912" alt="image" src="https://user-images.githubusercontent.com/10811515/60546297-eb0e4b00-9d14-11e9-8445-4357cd92bff6.png">

Atm that's my settings although might have changed since I last worked on xcresultviewer

Reply:
Looks like it might be a tabs vs spaces issue. 👀